### PR TITLE
Add quotes in log sampling config and add it in the stackdriver test

### PR DESCRIPTION
**Pull request by gargnupur**
Signed-off-by: gargnupur <gargnupur@google.com>

Please provide a description for what this PR is for.

And to help us figure out who should review this PR, please 
put an X in all the areas that this PR affects.

[ ] Configuration Infrastructure
[ ] Docs
[ ] Installation
[ ] Networking
[ ] Performance and Scalability
[ ] Policies and Telemetry
[ ] Security
[ ] Test and Release
[ ] User Experience
[ ] Developer Infrastructure

Pull Request Attributes

Please check any characteristics that apply to this pull request. 

[ ] Does not have any changes that may affect Istio users.

### CODE REVIEW

**Review by bianpengyuan — APPROVED** [review]

File: manifests/charts/istio-control/istio-discovery/templates/telemetryv2_1.6.yaml [hunk 1]
```diff
@@ -594,7 +594,7 @@ spec:
               config:
                 configuration: |
                   {
-                    "log_window_duration": {{ .Values.telemetry.v2.accessLogPolicy.logWindowDuration | quote }}
+                    "log_window_duration": "{{ .Values.telemetry.v2.accessLogPolicy.logWindowDuration }}"
```
Comment (richardwxn) [inline manifests/charts/istio-control/istio-discovery/templates/telemetryv2_1.6.yaml:597]: why not use quote function?

Comment (gargnupur) [reply manifests/charts/istio-control/istio-discovery/templates/telemetryv2_1.6.yaml:597]: it didn't work.. when I inspected envoyfilter for 1.6 too, quotes weren't there..

**Review by carolynhu — COMMENTED** [review]
@gargnupur thanks! I will trigger a new group of test after this PR is in.

**Review by richardwxn — APPROVED** [review]

**Pull request merged**